openshift/s2i/nginx-runtime/nginx.conf.template

```diff
@@ -40,7 +40,7 @@ http {
         ignore_invalid_headers off;
 
         # add in most common security headers
-        add_header Content-Security-Policy "default-src * data: blob: filesystem: 'unsafe-inline' 'unsafe-eval'";
+        add_header Content-Security-Policy "default-src 'self'; frame-ancestors 'self'; img-src 'self' data: https://openmaps.gov.bc.ca/ https://*.tile.openstreetmap.org/ https://server.arcgisonline.com/; frame-src 'self' https://*.oidc.gov.bc.ca; connect-src 'self' https://*.oidc.gov.bc.ca/ https://openmaps.gov.bc.ca/ https://server.arcgisonline.com/; form-action 'self'; style-src 'self' 'sha256-47DEQpj8HBSa+/TImW+5JCeuQeRkm5NMpJWZG3hSuFU=';";
```

Comment:
This is the actual CSP that runs when the app is deployed. Notice the sha1 has on the style-src attribute. Not ideal, but it appears that something within our app (I speculate a dependency) is using inline styles. I attempted to modify style tags within our application, and that did not result in a change to this hash.

Similarly, hoping to circle back to this if I get time. Would be nice to isolate the source of the inline script and remove it.

Reply:
I don't understand the sha statement?  Somewhere there is a request for `<link rel="stylesheet" src="sha...".>`?

Reply:
Yes Jeremy that's what I read from Devin's comment

Reply:
No, not quite. Let me provide some background links

Reply:
This is one of the articles I used to better understand this issue:
https://www.uriports.com/blog/creating-a-content-security-policy-csp/

Essentially to allow an inline style block, the csp must have a hash explicitly specified that corresponds to running a hash function on that block of inlined code.

Reply:
What that means is that if that inlined styling changes for any reason, the hashes will no longer match and the inlined style will be blocked. Assuming I'm right and the inlined styling is coming from a dependency, this would make upgrading packages more dangerous. As a result, I'd like to find and document the source of these inline styles, but I also have other stories this sprint and am not convinced this issue is the highest priority.